### ISSUE TITLE
Import for Enums does not work when using explicit enum name

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/ergo#700]
This works:
```
Welcome to ERGOTOP version 0.20.0-beta.2
ergo$ import org.accordproject.money.*
ergo$ return USD
Response. USD : CurrencyCode
ergo$ 
```

But this does not but it should:
```
Welcome to ERGOTOP version 0.20.0-beta.2
ergo$ import org.accordproject.money.CurrencyCode
ergo$ return USD
Type error (at line 1 col 7). Variable `USD' not found.
return USD
       ^^^
```

**Comment by jeromesimeon** [comment]
This is not specific to enums, but is true for important any constant:
```
bash-3.2$ cat test.ergo
namespace foo

define concept Foo{
	a: String
}
define constant pi = 3.14
bash-3.2$ ./bin/ergotop test.ergo 
Welcome to ERGOTOP version 0.20.0-beta.2
ergo$ import foo.pi
Compilation error (at line 1 col 7). Cannot import name 'pi' in CTO with namespace foo
import foo.pi
       ^^^^^^
ergo$ ^D
bash-3.2$ 
```

**Comment by jeromesimeon** [comment]
Fix in #704 Here is a test using ergotop:

<img width="1148" alt="Screenshot 2019-10-31 at 7 50 55 PM" src="https://user-images.githubusercontent.com/670099/67993581-cf11ac00-fc17-11e9-96ec-39f02c99210b.png">
